tox.ini

```diff
@@ -51,4 +52,4 @@ commands =
     npm config set spin false
     npm install
     npm run lint
-    npm run package
+    npm run sign
```

Comment:
Hmm, looks like `npm run sign` works on my dev machine, but even with the `passenv` line in `tox.ini` it seems like the env vars aren't getting seen.

Maybe revert this to `npm run package` for now? Packaging without signing is still useful for local dev, and will at least get us a sanity check on Travis as to whether the addon could be built.